src/routes/api/genre/genreController.js

```diff
@@ -0,0 +1,90 @@
+const uuidv4 = require("uuid/v4");
+const GenresModel = require("../../../models/GenresModel");
+
+class GenreController {
+  getGenres = async (req, res) => {
+    const results = await GenresModel.findAll();
+    if (results) {
+      res.status(200).json({ status: 200, genres: results });
+    } else {
+      res.status(403);
+    }
+  };
+
+  createGenre = (req, res) => {
+    const { title } = req.body;
+
+    GenresModel.create({
+      id: uuidv4(),
+      title,
+      createdAt: Date.now(),
+      updatedAt: Date.now()
+    })
+      .then(() => {
+        res.status(201).json({
+          msg: `${title} added successfully`
+        });
+      })
+      .catch(err => {
+        res.status(403).json({
```

Comment:
Should be `500` or internal server error, `403` means forbidden which is not the case here